README.md

```diff
@@ -11,6 +11,7 @@ A TypeScript library that converts GraphQL query and mutation strings into struc
 ## Features
 
 - ✅ **Full GraphQL Support**: Queries, mutations and subscriptions
+- ✅ **Inline Fragments**: Complete support for conditional type-based field selection
```

Comment:
This sounds like a nit, but since the readme is precious "advertising space": Could this please move down the list? I find it a less important feature than the other ones mentioned. I would move it possibly after "Aliases".